packages/dcos-integration-test/extra/test_ec2.py

```diff
@@ -145,7 +82,17 @@ def test_move_external_volume_to_new_agent(cluster):
         cluster.deploy_marathon_app(read_app, **deploy_kwargs)
         cluster.destroy_marathon_app(read_app['id'])
     finally:
-        try:
-            _delete_ec2_volume(test_label)
-        except Exception as ex:
-            raise Exception("Failed to clean up volume {}: {}".format(test_label, ex)) from ex
+        logging.info('Deleting '+test_label)
```

Comment:
This message might be unclear since there's nothing in `test_label` that indicates it's a volume name. `'Deleting volume ' + test_label` would be clearer.